### ISSUE TITLE
ClusterBlast in antismash 5 returns only bacterial hits for fungal genomes

### ISSUE THREAD

**Issue by MichaelFokinNZ** [antismash/antismash#147]
**Describe the bug**
ClusterBlast in antismash 5 returns **only bacterial hits** for fungal genomes

**Expected behavior**
ClusterBlast in antismash 4 returned **fungal hits** for fungal genomes

**To Reproduce**
For analysis issues, provide an accession number or a record fragment that can reproduce the problem. fungi-0d30459c-21b0-469a-84cf-146b64dcaebc

two screenshots attached for the same cluster in

 antismash4
![image](https://user-images.githubusercontent.com/7625343/57260854-8e4a3780-70b9-11e9-8fe4-79546d91760e.png)
and antismash5
!
![image](https://user-images.githubusercontent.com/7625343/57260865-97d39f80-70b9-11e9-8c0e-a9bfb62dba9a.png)

the problem applicable to all clusters :(

**Comment by kblin** [comment]
Yes, that is because we switched the ClusterBlast database to be derived from the antiSMASH database, and that doesn't contain fungal genomes at the moment. The sequence quality thresholds we set for our inputs meant that there were only around 25 fungal genomes that would have been included in the database, so we didn't follow that route.

We are planning on updating the antiSMASH database to antiSMASH 5.0 results in the second half of this year, and we will have another look at including some fungal genomes at that point, but a quick look at RefSeq tells me that there still only are around 50 high quality assemblies available.

I'm open to suggestions on how we can make ClusterBlast more useful for fungal genomes while not sacrificing the quality of data contained in the antiSMASH database.

**Comment by paulamool** [comment]
Hi Kblin, for people working with fungal genomes the clusterblast alignments to bacteria are a lot less informative than to fungi.  
I'm running the latest docker standalone CLI (5.0.0). As a work around, I  swapped V5.0.0 bacterial clusterblast data sets for the fungal cluster blast data sets V4.0.0 and get the following error. Is it possible to do this or is the code not backward compatible to older dbs? Any help would be greatly appreciated.

INFO     18/10 04:09:54   Running ClusterBlast
INFO     18/10 04:09:54   ClusterBlast: Loading gene cluster database into memory...
INFO     18/10 04:10:05   ClusterBlast: Loading gene cluster database proteins into memory...
INFO     18/10 04:11:46   Comparing regions to reference database
Traceback (most recent call last):
...
  File "/usr/local/lib/python3.5/dist-packages/antismash/modules/clusterblast/svg_builder.py", line 412, in <listcomp>
    proteins = [reference_proteins[protein] for protein in cluster.tags]
KeyError: ''
Running antiSMASH FAILED

**Comment by SJShaw** [comment]
When we rewrote most of the clusterblast code for version 5, we discovered that the older data had some issues such as referenced proteins being missing and containing two proteins with the same reference. Because of that, we updated the data to ensure it wasn't giving incorrect results, which also means that 5 is not compatible with the 4 data (without some changes to the data to remove those erroneous entries).
